enterprise/views.py

```diff
@@ -438,18 +440,23 @@ def post_course_specific_consent(self, request, course_id, consent_provided):
         except HttpClientError:
             raise Http404
 
-        enrollment_deferred = request.POST.get('enrollment_deferred')
```

Comment:
I can comment on `enrollment_deferred`; not on why it'd be in use/removed here. Essentially, `enrollment_deferred` is used in the Ecom flow when a user goes to redeem a coupon- it prevents a consent record from being recorded, but lets us use the consent UI anyway. It's assumed that if the user is then redirected back to `success_url`, any relevant consent requirement was met, and the overall process can proceed - in the Ecom flow, the consent record is created when the actual enrollment is.